rosdep/python.yaml

```diff
@@ -2451,6 +2451,36 @@ python-texttable:
 python-tftpy:
   fedora: [python-tftpy]
   ubuntu: [python-tftpy]
+python-theano:
+  arch: [python-theano]
+  debian: 
+    jessie:
+      pip:
+        packages: [Theano]
+    stretch: [python-theano]
+    sid: [python-theano]
```

Comment:
Please don't add sid. It's a rolling target so we don't try to define what's in it. This is also failing the CI due to being out of alphabetical order.